build/yamls/flow-visibility/base/clickhouse.yml

```diff
@@ -26,7 +26,17 @@ spec:
   defaults:
     templates:
       podTemplate: pod-template
+      serviceTemplate: service-template
   templates:
+    serviceTemplates:
+      - name: service-template
+        spec:
+          ports:
+            - name: http
+              port: 8123
+            - name: tcp
+              port: 9000
+          type: LoadBalancer
```

Comment:
I don't think this needs to be a LoadBalancer Service? It's not accessed from outside the cluster, it seems that the typical deployment at the moment is to deploy all the flow visibility stuff in the same cluster as the workloads. If the cluster doesn't support LoadBalancer Services, then the external IP will always show as pending.

Reply:
Thanks for addressing this. It makes sense for me to use ClusterIP for now. Updated.